src/amo/components/Errors/UnavailableForLegalReasons/index.js

```diff
@@ -0,0 +1,39 @@
+/* @flow */
+import * as React from 'react';
+import { compose } from 'redux';
+
+import ErrorComponent from 'amo/components/Errors/ErrorComponent';
+import translate from 'core/i18n/translate';
+import type { I18nType } from 'core/types/i18n';
+
+type Props = {||};
+
+type InternalProps = {|
+  ...Props,
+  i18n: I18nType,
+|};
+
+export class UnavailableForLegalReasonsBase extends React.Component<InternalProps> {
+  render() {
+    const { i18n } = this.props;
+
+    return (
+      <ErrorComponent
+        code={451}
+        header={i18n.gettext('Not Available In Your Region')}
```

Comment:
Should it say not available for legal reasons in your region?

Reply:
Sure, I can update it.